org.mixedrealitytoolkit.input/Editor/Inspectors/EyeGazeSimulationSettingsDrawer.cs

```diff
@@ -29,7 +29,7 @@ public class EyeGazeSimulationSettingsDrawer : PropertyDrawer
             SerializedProperty property,
             GUIContent label)
         {
-            return PropertyDrawerUtilities.CalculatePropertyHeight(6);
+            return PropertyDrawerUtilities.CalculatePropertyHeight(7);
```

Comment:
Magic number.  Can we at least add a const somewhere, to give the number a name?

Reply:
Refactored this out and added a doc comment.

Reply:
This is the pattern used by the pre-existing code and while I agree that MRTK should avoid 'magic numbers', i wouldn't enforce that level of change on a bug fix (other areas of the code are likely to be impacted).